### PR TITLE
feature: New page -> 404 Not Found

### PR DESCRIPTION
**Pull request by CodeMan99**
Includes prettier formatting of `src/assets/styles.css`

### CODE REVIEW

File: src/404.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: layouts/page.html
+title: 404 Not Found
+---
+
+## 404 No Bytes Found
+
+<code class="tty">
+    <pre><span class="tty-username">admin</span>&commat;<span class="tty-hostname">bytezone.dev</span> &dollar; ls -l -h /var/www/static</pre>
+    <pre>lrwxrwx--- 1 root www    9 Oct  4 09:58 <span class="tty-broken-link">current.html</span> &longrightarrow; <span class="tty-broken-link">page.html</span></pre>
+    <pre>-rw-r----- 1 root www 2.4K Oct  4 09:57 <a href="/">index.html</a></pre>
```
Comment (RMcNeely) [inline src/404.md]: I had no idea this was a link at all until I looked here.  We need to make this more clear somehow.

Comment (CodeMan99) [reply src/404.md]: The normal page navigation still exists. I included the href as an easter egg.

Comment (CodeMan99) [reply src/404.md]: Maybe just an underline?

Comment (CodeMan99) [reply src/404.md]: Added underline & silver text color. Which is a very slight color change.

File: src/assets/styles.css [hunk 2]
```diff
@@ -56,24 +56,119 @@ body {
...
+	color: red;
+}
+
+.git-diff-add-line::before {
+	content: "+";
+	color: #4e9a06;
+}
+
+.git-diff-add-line {
+	color: #4e9a06;
```
Comment (RMcNeely) [inline src/assets/styles.css]:
```suggestion
	color: #8dd946;
```

File: src/assets/styles.css [hunk 3]
```diff
@@ -56,24 +56,119 @@ body {
...
+	color: red;
+}
+
+.git-diff-remove-line {
+	color: red;
+}
+
+.git-diff-add-line::before {
+	content: "+";
+	color: #4e9a06;
```
Comment (RMcNeely) [inline src/assets/styles.css]:
```suggestion
	color: #8dd946;
```

File: src/assets/styles.css [hunk 4]
```diff
@@ -124,12 +124,16 @@ body {
 
 	& .tty-username {
 		color: #3465a4;
+		color: var(--lilac);
+		color: #06aeb1;
+		color: #d8b4dd;
```
Comment (CodeMan99) [inline src/assets/styles.css]: Why are there so many color properties here?

Comment (RMcNeely) [reply src/assets/styles.css]: Mostly so you can ask that question and see if you like them better :slightly_smiling_face: 

Ultimately if you're good with these values I want to remove them. If you check them they get successively lighter.

Comment (CodeMan99) [reply src/assets/styles.css]: Ah! Well, then... I choose `#06aeb1` (line 128). The next value `#d8b4dd` is really darn close to "skyblue", which is used in the git diff.

Comment (RMcNeely) [reply src/assets/styles.css]: Are you sure you're looking at this right? The current `#d8b4dd` is  light purple shade but `#06aeb1` is a light blue that's fairly close to the git diff color (that's why I changed it)

Comment (CodeMan99) [reply src/assets/styles.css]: VSCode tells me that "skyblue" is `#87ceeb`.

Comment (RMcNeely) [reply src/assets/styles.css]: Right, I wanted the username and hostname in the prompt to be a separate color from the cyan like blue of the diff, so I picked light purple.

Comment (CodeMan99) [reply src/assets/styles.css]: `#d8b4dd` is really darn close to `#87ceeb`. What am I missing?

There's almost no red in `#06aeb1`... Much larger difference to my eyes. Maybe increase the green value a little?

Comment (CodeMan99) [reply src/assets/styles.css]: PING!

**Comment by CodeMan99** [comment]
Merged as-is because it's been pending too dang long.

**Pull request merged**